server/services/interventionsService.test.ts

```diff
@@ -16,26 +16,27 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
     beforeEach(async () => {
       // This is just an example for getting Pact set up; we’ll properly design this endpoint later
       await provider.addInteraction({
-        state: 'There is an existing referral with ID of 1',
+        state: 'There is an existing referral with known id',
         uponReceiving: 'a request for that referral',
         withRequest: {
           method: 'GET',
-          path: '/referrals/1',
+          path: '/draft-referral/ac386c25-52c8-41fa-9213-fcf42e24b0b5',
           headers: { Accept: 'application/json' },
         },
         willRespondWith: {
           status: 200,
           body: Matchers.like({
-            id: '1',
+            id: 'ac386c25-52c8-41fa-9213-fcf42e24b0b5',
+            created: '2020-12-02 17:20:14',
```

Comment:
The addition of `created` doesn't seem related to the rest of the changes here, should it be here? Also what format is this datetime? It's not a valid ISO 8601 value.

Reply:
will remove and add in a seperate PR. good point on ISO formatting. it's the default serialization of java `Date`. i sort of figured it would be ISO formatting and did not notice it wasn't.